src/core/insights/command-queue.ts

```
@@ -0,0 +1,136 @@
+        this.realImplementation &&
+        typeof this.realImplementation[cmd.methodName] === "function"
+      ) {
+        try {
+          if (this.debugMode) {
+            logger.debug(
+              `Executing queued command: ${cmd.methodName}`,
+              cmd.args,
+            );
+          }
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-function-type
+          (this.realImplementation[cmd.methodName] as Function)(
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)